libs/sdk-backend-tiger/src/convertors/toBackend/afm/toAfmResultSpec.ts

```diff
@@ -27,21 +33,18 @@ function convertAFM(def: IExecutionDefinition): ExecuteAFM.IAfm {
     };
 }
 
-function convertResultSpec(def: IExecutionDefinition): ExecuteAFM.IResultSpec {
+function convertResultSpec(def: IExecutionDefinition): ResultSpec {
     const convertedDimensions = convertDimensions(def);
-    const dimsProp = !isEmpty(convertedDimensions) ? { dimensions: convertedDimensions } : {};
 
-    return {
-        ...dimsProp,
-    };
+    return { dimensions: convertedDimensions };
```

Comment:
we were previously stripping away empty dim array. is this not allowed according to the spec & thus now removed? (I see tests have changed to include empty dims so i wonder)

Reply:
yes, `dimensions` are required, it can be empty array